x-pack/plugins/cases/public/components/add_comment/index.test.tsx

```diff
@@ -79,8 +79,8 @@ describe('AddComment ', () => {
     });
 
     fireEvent.click(screen.getByTestId('submit-comment'));
+    await waitFor(() => expect(screen.getByTestId('loading-spinner')).toBeInTheDocument());
```

Comment:
Why this change? We don't usually need waitFor for queries that can be done with `find*` which already uses `waitFor` under the hood.